app/directives/bucket-page-status-card/bucket-page-status-card.html

```diff
@@ -101,6 +101,16 @@ <h3 class="bucket-page__status">Paid</h3>
           <p class="bucket-page__status-description"><em>This bucket was marked as paid on {{ bucket.paidAt | exactDateWithTime }}</em></p>
         </div>
       </div>
+
+      <md-button
+        ng-if="userCanUnarchiveBucket()"
+        class="md-raised md-primary bucket-page__start-funding-btn"
+        ng-click='unarchive()'
```

Comment:
i think the CSS class on this is wrong, it should be something like `bucket-page__unarchive-btn` right?